_data/banking.yml

```diff
@@ -381,6 +381,13 @@ websites:
       hardware: Yes
       doc: http://www.intesasanpaolo.com/servizi-multicanale/sicurezza.jsp
 
+    - name: La Banque Postale
+      url: https://www.labanquepostale.fr/
+      twitter: labanquepostale
+      facebook: labanquepostale
+      img: labanquepostale.png
+      tfa: No
+
```

Comment:
Hello @divayprakash,
Would you also be able to add the following line: `lang: fr` please?